src/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -839,7 +910,114 @@ public override void ProcessEvents()
                                 break;
                             }
                         }
-                        
+                        else if (e.ClientMessageEvent.message_type == _atom_xdnd_enter)
+                        {
+                            // Xdnd started
+                            bool useList = ((e.ClientMessageEvent.ptr2.ToInt64() & 1) == 1);
+                            sourceHandler = e.ClientMessageEvent.ptr1;
+                            sourceXdndVersion = e.ClientMessageEvent.ptr2.ToInt64() >> 24;
```

Comment:
ToInt64 because in Xlib.h it defined like long[5]. 
```
typedef struct {
    int type;
    unsigned long serial;   /* # of last request processed by server */
    Bool send_event;    /* true if this came from a SendEvent request */
    Display *display;   /* Display the event was read from */
    Window window;
    Atom message_type;
    int format;
    union {
        char b[20];
        short s[10];
        long l[5];
    } data;
} XClientMessageEvent;
```